Towny/src/main/java/com/palmergames/bukkit/towny/tasks/TownClaim.java

```diff
@@ -56,7 +57,7 @@ public class TownClaim implements Runnable {
 	 * @param forced admin forced claim/unclaim
 	 * @param isOutpost if claim is/was an Outpost   
 	 */
-	public TownClaim(Towny plugin, Player player, Town town, List<WorldCoord> selection, boolean isOutpost, boolean claim, boolean forced) {

```

Comment:
Instead of adding this to the constructor, I would make a second constructor that adds a isOverClaim option.

The existing constructor just calls the new constructor is a false isOverClaim.

This results in no changes to the codes required for most of the code base, and for any other plugins that might be using the task their code will not break from a NoSuchMethodError.